tests/Utility/OrganizationFeaturesTest.php

```diff
@@ -42,6 +44,34 @@ public function testOrganizationFeatures(?string $propertyValue, bool $expectedR
         $this->assertEquals($expectedResult, OrganizationFeatures::isMonetizationEnabled($organization));
     }
 
+    /**
+     * Test if an organization has pagination enabled.
+     */
+    public function testPaginationEnabled(): void
+    {
+        /** @var \Apigee\Edge\Api\Management\Entity\OrganizationInterface $organization */
+        $organization = $this->getMockBuilder(OrganizationInterface::class)->getMock();
+        $organization->method('getPropertyValue')->will($this->returnValueMap([
+            ['features.isCpsEnabled', null],
+            ['features.hybrid.enabled', 'true'],
+        ]));
+        $this->assertTrue(OrganizationFeatures::isPaginationEnabled($organization));
+
+        $organization = $this->getMockBuilder(OrganizationInterface::class)->getMock();
+        $organization->method('getPropertyValue')->will($this->returnValueMap([
+            ['features.isCpsEnabled', 'true'],
+            ['features.hybrid.enabled', null],
+        ]));
+        $this->assertTrue(OrganizationFeatures::isPaginationEnabled($organization));
+
+        $organization = $this->getMockBuilder(OrganizationInterface::class)->getMock();
+        $organization->method('getPropertyValue')->will($this->returnValueMap([
+            ['features.isCpsEnabled', 'false'],
```

Comment:
Please introduce a data provider, just like I did for `testOrganizationFeatures`, that will reduce the code duplication in the test method.

Reply:
Refactored test :+1: